app/sensors/DataSearch.js

```diff
@@ -462,11 +462,14 @@ export default class DataSearch extends Component {
 				value: this.state.currentValue
 			});
 		}
+		if (this.props.onBlur) this.props.onBlur(event);
 	}
 
 	handleKeyPress(event) {
 		if (event.key === "Enter") {
 			event.target.blur();
+		} else {
```

Comment:
The native events should always execute when present. So you can remove `else` block from here.

Reply:
Updated.